tests/Controller/DashboardStatsControllerTest.php

```diff
@@ -74,7 +74,7 @@ public function test_paused_status_is_reflected_if_all_master_supervisors_are_pa
         $masters = Mockery::mock(MasterSupervisorRepository::class);
         $masters->shouldReceive('all')->andReturn([
             (object) [
-                'status' => 'running',
+                'status' => 'paused',
```

Comment:
you may want to change the method name to `test_paused_status_is_reflected_if_any_master_supervisor_is_paused`